src/test/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreTests.java

```diff
@@ -1451,6 +1454,110 @@ public void testSnapshotMoreThanOnce() throws ExecutionException, InterruptedExc
         }
     }
 
+    @Test
+    public void changeSettingsOnRestoreTest() throws Exception {
+        Client client = client();
+
+        logger.info("-->  creating repository");
+        assertAcked(client.admin().cluster().preparePutRepository("test-repo")
+                .setType("fs").setSettings(ImmutableSettings.settingsBuilder()
+                        .put("location", newTempDirPath(LifecycleScope.SUITE))
```

Comment:
Why does the temp dir need suite scope here? It looks like it doesn't escape this test so it can use the default TEST scope?